app/views/application/error.js.erb

```diff
@@ -1,12 +1,29 @@
 (() => {
-  const element = document.createElement("div")
-  element.classList.add("alert")
-  element.classList.add("alert--error")
-  element.innerText = "<%= @message || "Something went wrong when performing that action." %>"
+  const textNode = document.createElement("p")
+  textNode.classList.add("mt-1/16", "mb-1/16")
+  textNode.innerText = "<%= @message || 'Something went wrong when performing that action.' %>"
+
+  const closeButton = document.createElement("button", { "data-role": "dismiss-parent" })
+  closeButton.classList.add("button", "p-0", "pl-1/16", "pr-1/16", "text-white")
+  closeButton.dataset.role = "dismiss-parent"
+  closeButton.innerText = "✕"
+
+  function dismissParent(event) {
+    const parent = this.parentNode
+    parent.classList.add("fade-out")
+    setTimeout((parent) => {
+      parent.remove()
+    }, 500, parent)
```

Comment:
What does `parent` do here?

Reply:
The timeout function takes in one argument, a node to remove when run, and the `setTimeout` function can pass the third argument onwards in as an argument to the provided function. However, I forgot that JavaScript will include the parent function scope in the closure for the arrow function, so this is entirely unnecessary.

Reply:
Ah cool, didn't know it could take more arguments!